gpustat/core.py

```diff
@@ -181,6 +181,7 @@ def print_to(self, fp,
                  show_codec="",
                  show_power=None,
                  gpuname_width=None,
+                 no_processes=False,
```

Comment:
We've got some conflicts. Can you please move this argument near to `show_full_cmd` (for better locality), and *squash* all the commits without merge-with-up-stream merge-commits? Thank you.

Reply:
squash merging is activated, lmk if it works :)

Reply:
I can do squash merging once LGTM, but it'd be also helpful for you to learn an even better github PR workflow with rebasing and squashing commits. Anyway thanks for the update!